css/main.css

```diff
@@ -754,7 +757,69 @@ h2 {
 }
 
 #footer a:hover {
-  color:#1B7B98
+  color:#05cc47
+}
+
+
+/* Added By Aakash
+The Flash Page CSS*/
+
+body, html {
+  height: 100%;
+}
+
+.courosal {
+  animation-name: yo;
+  animation-duration: 10s;
+  animation-iteration-count: infinite;
+  transition: all .5s;
+  height: 100%;
+  width: 100%;
+  background-size: cover;
+  z-index: +1000;
+  -webkit-backface-visibility: hidden;
+  -moz-backface-visibility:    hidden;
+  -ms-backface-visibility:     hidden;
+  backface-visibility: hidden;
+  -webkit-transform: translate3d(0);
+  transform: translate3d(0);
+}
+
+@keyframes yo {
```

Comment:
Can this name be more descriptive?
So that anybody who is reading your code will understand it.